.changes/1.23.0.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "category": "``gamelift``",
+    "description": "[``botocore``] Documentation updates for Amazon GameLift.",
+    "type": "api-change"
+  },
+  {
+    "category": "``mq``",
+    "description": "[``botocore``] This release adds the CRITICAL_ACTION_REQUIRED broker state and the ActionRequired API property. CRITICAL_ACTION_REQUIRED informs you when your broker is degraded. ActionRequired provides you with a code which you can use to find instructions in the Developer Guide on how to resolve the issue.",
+    "type": "api-change"
+  },
+  {
+    "category": "IMDS",
```

Comment:
Just realized this isn’t in the updates. Let’s either remove the change log entry or add the IMDS changes.